setup/installation.md

```diff
@@ -75,9 +75,10 @@ kubectl config set-context --current --namespace=coder
 
    _Note: If you omit `--version`, you'll upgrade to the latest version._
 
-   > We do not provide documentation for release candidates (any version containing
-   > `-rc`), and they should not be used unless you've been instructed to use them
-   > by Coder.
+   > We do not provide documentation for release candidates, and you should not
+   use them unless you've been instructed to do so by Coder. You can identify
+   release candidates by the presence of `-rc` in the version number (e.g.,
+   `1.16.0-rc.1`).
```

Comment:
We should clarify that omitting `--version` will not attempt to use a release candidate unless you also specify `--devel`.